src/TraceEvent/Computers/StartStopActivityComputer.cs

```diff
@@ -74,8 +74,8 @@ public StartStopActivityComputer(TraceLogEventSource source, ActivityComputer ta
 
             Dictionary<long, int> mapConnectionToTargetProcess = new Dictionary<long, int>();
 #endif 
-            var dynamicParser = source.Dynamic;
-            dynamicParser.All += delegate (TraceEvent data)
+            //var dynamicParser = source.Dynamic;
```

Comment:
My recommendation is to use the source.AddCallbackForProviderEvents(CouldBeStartStop, callback) to create a filter (CouldBeStartStop) that includes events with 'Start' or 'Stop' suffixes, or are from a provider that is special cased in the code).